spec/factories/activity.rb

```diff
@@ -100,15 +104,28 @@
     planned_end_date { nil }
     actual_start_date { nil }
     actual_end_date { nil }
+    geography { nil }
     recipient_region { nil }
+    recipient_country { nil }
     flow { nil }
     finance { nil }
     aid_type { nil }
     tied_status { nil }
   end
 
-  trait :at_country_step do
```

Comment:
Do we need an `at_country_step`, too? Does this imply that we're not testing that case?

Reply:
I don't think it implies that. This change is part of the first refactor I did. We originally stored `recpient_region` but asked for country – I wanted it to all be region to make way for also being able to ask for country.

I don't believe we explicitly test for how the steps behaviour for every step, it just happend that we wrote the test for an 'incomplete' wizard from using the country step, see:

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/ce530674dc8d497b0d1876852d8a3f7e711349f7/spec/features/staff/users_can_edit_an_activity_spec.rb#L56